redbot/cogs/cleanup/cleanup.py

```diff
@@ -271,6 +271,40 @@ def check(m):
 
         await mass_purge(to_delete, channel)
 
+    @cleanup.command()
+    @commands.guild_only()
+    @commands.bot_has_permissions(manage_messages=True)
+    async def between(
+        self, ctx: commands.Context, one: int, two: int, delete_pinned: bool = False
+    ):
+        """Delete the messages between Messsage One and Message Two, providing the messages IDs.
+
+        The first message ID should be the older message, and the second one the newer.
+
+        Example:
+            `[p]cleanup between 123456789123456789 987654321987654321`
+        """
+        channel = ctx.channel
+        author = ctx.author
+        try:
+            mone = await channel.fetch_message(one)
+        except discord.errors.Notfound:
+            return await ctx.send(f"Could not find a message with the ID of {one}.")
+        try:
+            mtwo = await channel.fetch_message(two)
+        except discord.errors.Notfound:
+            return await ctx.send(f"Could not find a message with the ID of {two}.")
```

Comment:
Same here on the f-strings and i18n

Reply:
Addressed in 4bc6b5e0ff34cc458f92b7a05f3981e54ff9de48.